client/modules/wj-css/src/base/_a11y.scss

```diff
@@ -0,0 +1,39 @@
+// Makes tags more semantic and meaningful for screen-readers.
+mark {
+  @include label-for-screenreaders("highlighted text");
+}
+s {
+  @include label-for-screenreaders("stricken text");
+}
+s {
+  @include label-for-screenreaders("deleted text");
+}
+ins {
+  @include label-for-screenreaders("inserted text");
+}
+// Change the cursor on busy elements.
+[aria-busy="true" i] {
+  cursor: progress;
+}
+
+// Change the cursor on control elements.
+[aria-controls] {
+  cursor: pointer;
+}
+
+// Change the cursor on disabled, not-editable, or otherwise inoperable elements.
+[aria-disabled="true" i],
+[disabled] {
+  cursor: not-allowed;
+}
+
+// Change the display on visually hidden accessible elements.
+
+[aria-hidden="false" i][hidden] {
+  display: initial;
+}
+
+[aria-hidden="false" i][hidden]:not(:focus) {
+  position: absolute;
+  clip: rect(0, 0, 0, 0);
+}
```

Comment:
When would these situations occur?

Reply:
Great question - this actually comes from `normalize.css`. If I had a guess, these selectors are for elements that can be tab focused but aren't normally visible. A good example of that is on Github, just open a new page on Github and press tab. It'll reveal a "jump to main content" button on the top left.